README.md

```diff
@@ -3,88 +3,85 @@
 
 ## General
 
-* When building interface accessible by other apps/users, try stick to standards and schemas (e.g. swagger). Make the schemas discoverable, reference in README.md on oda-hub repositories.
-* Follow [issue handling principles](https://github.com/oda-hub/doc-ops-reporting#issue-handling-principles) - in short, do not expect user issues to be left unattended, never.
-* Do not be afraid to introduce large changes, but only if they respect previous interfaces
+* When building interfaces accessible by other apps/users, try and stick to a pre-defined set of standards and schemas (e.g. swagger). Make the schemas discoverable, ideally referenced in the README.md on oda-hub repositories.
+* Follow the [issue handling principles](https://github.com/oda-hub/doc-ops-reporting#issue-handling-principles) - in order to never have user issues unhandled.
+* Do not be afraid to introduce large changes, but only if they respect previous interfaces.
 
 ## Do not forget to
 
-we all do many things, but we should be concious that some actions are needed for smooth progress of collective project.
-Even though these actions may be delayed as necessary, it should be done with intent.
-Too much notifications and distractions are detrimental to productivity, but one should aim keep some level of engagement.
+We all do many things, but we should be concious that certain actions are crucial for the progress of collective projects: even though those may be delayed if necessary, there should always be a reason behind. Too much notifications and distractions are detrimental to productivity, but some level of engagement should  always be maintaned.
 
-It may be advisable to make sure all actions and channels converge to email. 
+It may be advisable to make sure all actions and channels converge to email and notifications, in order to keep everybody aware of the ongoing changes:
```

Comment:
I also tend to be present as much as possible on many channels beyond mails (and phone of course), but I agree that this is not the case for everybody.

And I thought that providing other notifications systems would help in keeping people more aware of what is going on, but it is important to have the mails as the main one since is the official channel, that everyone involved will read for sure.